test/test_files/tinySNB/function/string.test

```diff
@@ -318,6 +339,13 @@ Faroo
 <Greg
 Huber
 
+-NAME LpadUTF8String
+-QUERY MATCH (m:movies) RETURN lpad(m.name, 25, "âó")
+---- 3
+âóâóâóâSóló cón tu párejâ
+âóâThe 😂😃🧘🏻‍♂️🌍🌦️🍞🚗 movie
+âóâóâóâóâóâóâóâóâóâóâRoma
+
```

Comment:
Can you add tests for lpad/rpad negative idx as well?

Reply:
done